lib/macmillan/utils/rspec/matchers/be_valid_mosaic.rb

```diff
@@ -0,0 +1,43 @@
+require 'rspec/expectations'
+
+### Example:
+### it { is_expected.to be_valid_mosaic }
+#
+### example output:
+#
+### Failure/Error: it { is_expected.to be_valid_mosaic }
+### Missing required property: template in /layout
+
+RSpec::Matchers.define :be_valid_mosaic do
```

Comment:
Yes valid point, didn't realise that this repo was public :) I just wanted this shared between projects to ensure we are all validating against a common source of truth (mojson). 

Just thinking about this and actually it might not work, since it needs the mojson binary on jenkins 
:( may need to think about this one.